### PR TITLE
Fix eagerload's default value in JHipster v7.7.0

### PR DESCRIPTION
**Pull request by mraible**

### CODE REVIEW

**Comment by mraible** [comment]
I'm looking into the failures now. My guess is there might be a situation where `eagerload` is false by default.

File: generators/server/generator.mjs [hunk 1]
```diff
@@ -351,7 +351,11 @@ class `
             )
             .replaceAll(
               `@RequestParam(required = false, defaultValue = "false") boolean eagerload`,
-              `@RequestParam(name = "eagerload",required = false, defaultValue = "false") boolean eagerload`
+              `@RequestParam(name = "eagerload", required = false, defaultValue = "false") boolean eagerload`
```
Comment (mshima) [inline generators/server/generator.mjs:354]: I will probably change this at generator-jhipster too
```suggestion
              `@RequestParam(name = "eagerload", required = false, defaultValue = "true") boolean eagerload`
```

File: generators/server/generator.mjs [hunk 2]
```diff
@@ -351,7 +351,11 @@ class `
             )
             .replaceAll(
               `@RequestParam(required = false, defaultValue = "false") boolean eagerload`,
-              `@RequestParam(name = "eagerload",required = false, defaultValue = "false") boolean eagerload`
+              `@RequestParam(name = "eagerload", required = false, defaultValue = "true") boolean eagerload`
```
Comment (mraible) [inline generators/server/generator.mjs:354]: @mshima Why do we need to replace `defaultValue = "false"` with `defaultValue = "true"`? If the default should be true, shouldn't that be done in JHipster core?

Comment (mshima) [reply generators/server/generator.mjs:354]: Yes I will do that.

Comment (mshima) [reply generators/server/generator.mjs:354]: I think you are right, let’s keep the same behavior.
```suggestion
              `@RequestParam(name = "eagerload", required = false, defaultValue = "false") boolean eagerload`
```

**Pull request merged**